### PR TITLE
ContainerInfo API

### PR DESCRIPTION
**Pull request by cgtexmex**
This provides the initial implementation of container info and will return the container information available to the portlayer.  

Much like the current implementation of containerlistinfo, this API will query the infrastructure for accurate container state / attribute information.

Fixes # 1393

### CODE REVIEW

**Comment by jzt** [comment]
LGTM

File: lib/apiservers/portlayer/swagger.yml [hunk 1]
```diff
@@ -1082,12 +1082,14 @@ definitions:
...
+      layerId:
         type: string
-      created:
+      repoName:
         type: string
-        format: datetime
-      name:
+      created:
+        type: integer
+        format: int64
```
Comment (sflxn) [inline lib/apiservers/portlayer/swagger.yml]: What's the interpretation of this date value?  Or better yet, how do I turn it into a DateTime?

File: lib/portlayer/exec/container.go [hunk 2]
```diff
@@ -250,6 +251,51 @@ func (c *Container) Remove(ctx context.Context) error {
 	return nil
 }
 
+// Grab the info for the requested container
+// TODO:  Possibly change so that hanlder requests a handle to the
```
Comment (caglar10ur) [inline lib/portlayer/exec/container.go]: [cosmetic] s/hanlder/handler/g

**Comment by caglar10ur** [comment]
LGTM :sheep: :it:

File: lib/apiservers/portlayer/swagger.yml (hunk 1 above)
Comment (cgtexmex) [inline lib/apiservers/portlayer/swagger.yml]: That's the unix time -- it's placed in the vmx file on container vm creation.  Using time.Unix(n,n) you can covert to time.  i.e. https://play.golang.org/p/Dx2asGut5Y

**Comment by sflxn** [comment]
Some of these changes to the swagger spec break PR #1307, but we're delaying issue #370 anyway so I can update that PR to reflect your swagger changes.

**Comment by caglar10ur** [comment]
@sflxn I have #1307 working here on top of this as I'm testing docker-compose integration :)

**Comment by cgtexmex** [comment]
rebase

**Pull request merged**